python_modules/dagster/dagster/_daemon/sensor.py

```diff
@@ -1316,16 +1319,21 @@ def _get_or_create_sensor_run(
         else:
             logger.info(
                 f"Run {run.run_id} already created with the run key "
-                f"`{run_request.run_key}` for {remote_sensor.name}"
+                f"`{run_key}` for {remote_sensor.name}"
             )
             return run
```

Comment:
just confirming that this log line still fires in this case

Reply:
yes, I just pulled out `run_key` into a separate var for typing reasons